testdata/struct_comment/web/handler.go

```diff
@@ -18,5 +18,5 @@ type Post struct {
 type APIError struct {
 	// Error an Api error
 	Error     string
```

Comment:
please add line comment. because I want to check no change.
eg: `Error     string // This is  error`

Reply:
ok